mission-11/docker-compose.yaml

```
@@ -0,0 +1,33 @@
+version: '3.8'
+
+services:
+  app:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - "3000:3000"
+    volumes:
+      - ./uploads:/app/uploads # Local folder bind mount for file uploads
+    environment:
+      DATABASE_URL: postgresql://user:password@db:5432/mydatabase?schema=public
+      JWT_SECRET_KEY: your_jwt_secret_key
+      JWT_REFRESH_SECRET_KEY: your_jwt_refresh_secret_key
+      PORT: 3000
+      NODE_ENV: development
```

```suggestion
    environment:
      DATABASE_URL: postgresql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@db:5432/${POSTGRES_DB}?schema=public
      JWT_SECRET_KEY: ${JWT_SECRET_KEY}
      JWT_REFRESH_SECRET_KEY: ${JWT_REFRESH_SECRET_KEY}
    env_file:
      - .env
```

Git 히스토리에 보안 취약점을 남기는 것은 좋지 않습니다.
물론 DB를 인터널하게 관리한다면 옵셔널로 분리할 수도 있을 것 같아요.